convert2rhel/unit_tests/subscription_test.py

```diff
@@ -333,3 +352,45 @@ def test_check_needed_repos_availability(self):
     def test_check_needed_repos_availability_no_repo_available(self):
         subscription.check_needed_repos_availability(["rhel"])
         self.assertTrue("rhel repository is not available" in logger.CustomLogger.warning.msg)
+
+    class VersionMocked(unit_tests.MockFunction):
+        def __init__(self, version):
+            self.version = version
+
+        def __call__(self, version):
+            return self.version
+
+    @unit_tests.mock(subscription.logging, "getLogger", GetLoggerMocked())
+    @unit_tests.mock(utils, "parse_version_number", VersionMocked(version={'major': '6', 'minor': '6'}))
```

Comment:
Be true to what the mocked function would return
```suggestion
    @unit_tests.mock(utils, "parse_version_number", VersionMocked(version={'major': '6', 'minor': '6', release: None}))
```